pkgs/tools/admin/trivy/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "trivy";
+  version = "0.11.0";
+
+  src = fetchFromGitHub {
+    owner = "aquasecurity";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0mqzs0yjlqpsn2540kgl4gn77qjg3kfls1zwgjfxy66lazic3f1h";
+  };
+
+  vendorSha256 = "1jmmx83c0qbqyppx0w8d9djz2j2i7dgibq9lb3vhdz78wc40zywy";
+
+  buildFlagsArray = [
+    "-ldflags="
+    "-s"
+    "-w"
+    "-X main.version=v${version}"
+  ];
+
+  meta = with stdenv.lib; {
+    description = "A Simple and Comprehensive Vulnerability Scanner for Containers, Suitable for CI.";
+    longDescription = ''
+      Trivy is a simple and comprehensive vulnerability scanner for containers and other artifacts.
+      A software vulnerability is a glitch, flaw, or weakness present in the software or in an Operating System.
+      Trivy detects vulnerabilities of OS packages (Alpine, RHEL, CentOS, etc.) and application dependencies (Bundler, Composer, npm, yarn, etc.).
```

Comment:
Try to write the longDescription lines within the 80 characters limit.

Reply:
Kk will update tomorrow.

Cut off at 80 exactly or allow the current word over?

E.g.

```
             V
Some words here
to demo

Vs
             V
Some words
here to demo
```

Reply:
Hard limit. Cut it off at 80.

Reply:
Fixed.
Is it worth changing this since the linux kernel has bumped from 80 to 100?

Reply:
Linux is a beast on its own. It isn't a fail-proof example of practices to be universally replicated. (I need to dig deeper on this recent change on the limit, tho.)

But I don't think it doesn't matter here. Linux is (mostly) C code. The `description` and `longDescription` are English prose. Readability is more important in this particular case, and I don't think we can assume everyone is typing on a widescreen monitor and with widescreen eyes. Not yet :)